src/components/Schedule/NewServiceForm.tsx

```diff
@@ -62,7 +63,7 @@ export const NewServiceForm = ({ onSubmit, onClose, error }: NewServiceFormProps
       <h2>Add A New Event</h2>
       {error && (
         <div style={{ color: 'red' }}>
-          {`Status code ${error.status}: ${error.message}`}
+          {`Status code ${error.response.status}: ${error.response.statusText}`}
```

Comment:
NewScheduleForm.tsx uses optional chaining, so I'd assume we should here too:
```suggestion
          {`Status code ${error?.response.status}: ${error?.response.statusText}`}
```

Reply:
Technically the previous method didn't need optional chaining as `error &&` handles it, but also in the previous method, I've made comments about using `error.response.{key}` at all